lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -120,3 +131,6 @@ def _set_log_level(self, options):
         elif options.get('verbosity') >= 1:
             log_level = logging.INFO
         log.setLevel(log_level)
+
+    def _latest_settings(self):
+        return ComputeGradesSetting.objects.order_by('-id')[0]
```

Comment:
@sanfordstudent Please update this to `ComputeGradesSetting.current`.  That is the standard and performant way of getting the most recent setting.  See https://github.com/edx/django-config-models/blob/master/config_models/models.py#L131  You'll notice that the current setting is cached for later retrieval as well.